playbooks/roles/machine/tasks/install_machine_agent_linux.yml

```diff
@@ -158,37 +158,5 @@
 
 
   #Custom Extension Prep
-  - name: Create subdirectory for custom extension
-    file:
-      path: "{{ machine_agent_dest_folder_linux }}/monitors/customextensions"
-      state: directory
-      mode: 0755
-
-  #  - name: Download custom extension 
-      #  get_url:
-      #     url: '{{ af_env }}/{{repo}}/{{extension_file}}'
-      #     dest: "{{ machine_agent_dest_folder_linux }}/monitors/customextensions"
-      #     username: "{{ username }}"
-      #     password: "{{ password }}"
-      #     force: true
-      #   no_log: true #To prevent passwords to be on logs
-      #   changed_when: false
-      #   register: result
-      #   failed_when: result.status_code != 200
-
-    #  - name: Unzip the Machine agent file
-    # unarchive:
-    #   src:  "{{ machine_agent_dest_folder_linux }}/monitors/customextensions"
-    #   dest: "{{ machine_agent_dest_folder_linux }}/monitors/customextensions"
-    #   owner: "{{ appdynamics_user }}"
-    #   group: "{{ appdynamics_user }}"
-    #   mode: 0755
-    #   remote_src: yes
-    # changed_when: false
-
-  #  - name: Install monitor.xml file
-  #   file:
-  #     path: ""
-  #     state: absent
-  #   changed_when: false
-
+  - include_tasks: configure_custom_extensions_linux.yml
```

Comment:
These new. lines I would remove from here completely and start in a new branch for extensions.
Here,  configure_custom_extensions_linux.yml does not exist

Reply:
Okay will update on this branch now